site/it/tutorials/customization/basics.ipynb

```diff
@@ -1,480 +1,480 @@
+        "colab": {},
+        "colab_type": "code",
+        "id": "vjBPmYjLdFmk"
+      },
+      "outputs": [],
+      "source": [
+        "import tensorflow as tf"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "H9UySOPLXdaw"
+      },
+      "source": [
+        "## Tensori\n",
+        "\n",
+        "Un Tensore è un array multi-dimensionale. Similmente all'oggetto `ndarray` di NumPy, gli oggetti `tf.Tensor` hanno un tipo di dato e una forma. Inoltre, gli oggetti `tf.Tensor` possono risiedere in un acceleratore di memoria (come una GPU). TensorFlow offre una libreria ricca di operazioni ([tf.add](https://www.tensorflow.org/api_docs/python/tf/add), [tf.matmul](https://www.tensorflow.org/api_docs/python/tf/matmul), [tf.linalg.inv](https://www.tensorflow.org/api_docs/python/tf/linalg/inv) etc.) che consumano e producono oggetti `tf.Tensor`. Queste operazioni convertono automaticamente tipi nativi di Python, ad esempio:\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "cellView": "code",
+        "colab": {},
+        "colab_type": "code",
+        "id": "ngUe237Wt48W"
+      },
+      "outputs": [],
+      "source": [
+        "print(tf.add(1, 2))\n",
+        "print(tf.add([1, 2], [3, 4]))\n",
+        "print(tf.square(5))\n",
+        "print(tf.reduce_sum([1, 2, 3]))\n",
+        "\n",
+        "# Operator overloading is also supported\n",
+        "print(tf.square(2) + tf.square(3))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "IDY4WsYRhP81"
+      },
+      "source": [
+        "Ogni oggetto `tf.Tensor` ha una forma e un tipo di dato:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "srYWH1MdJNG7"
+      },
+      "outputs": [],
+      "source": [
+        "x = tf.matmul([[1]], [[2, 3]])\n",
+        "print(x)\n",
+        "print(x.shape)\n",
+        "print(x.dtype)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "eBPw8e8vrsom"
+      },
+      "source": [
+        "Le differenze più ovvie tra gli array NumPy e i `tf.Tensor` sono:\n",
+        "\n",
+        "1. I tensori possono essere supportati da acceleratori di memoria (come GPU, TPU).\n",
+        "2. I tensori sono immutabili."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Dwi1tdW3JBw6"
+      },
+      "source": [
+        "### Compatibilità con NumPy \n",
+        "\n",
+        "La conversione tra oggetti `tf.Tensor` di TensorFlow e un `ndarray` di NumPy è facile:\n",
+        "\n",
+        "* Le operazioni di TensorFlow convertono automaticamente gli ndarrays di NumPy a tensori.\n",
```

Comment:
Le operazioni di TensorFlow ... a tensori. --> Le operazioni di TensorFlow ... in tensori.

Reply:
This change is ok with me